### DISCUSSION TITLE
Modifying space around characters

### DISCUSSION THREAD

**Discussion by eboston** [fontforge/fontforge#4864] [Q&A]
I am working on a project for a Nextion display and have a font I would like to use.  The problem I am having is there is a lot of whitespace above the character and the way the display works is it forces this area to be shown.  As a result, there is a lot of blank space I can't utilize.

Can FontForge be used to edit the font in a way to remove this excess space?

**Comment by frank-trampe** [comment]
The font likely has an unnecessarily large ascent value. If you open it in FontForge, you can adjust it easily in the Font Info view.

**Reply by eboston** [reply]
I opened the font and selected Element/Font Info.  Moved to the General option on the left and changed the Ascent value from 1638 to 1500.  Looking at the outline window of the '0', I see the line showing what I think is the acsent line changed from the 1638 it was to the 1500.  However, the height of the character also changed so there is still the same distance between that line and the character.  I'm guessing I must be doing something wrong.  Any suggestions?

**Reply by eboston** [reply]
I think I got it.  I found some info at http://designwithfontforge.com/en-US/Line_Spacing.html that talks about changing the line height.  I followed that and the new font is a lot closer to the top than the old one.

**Reply by eboston** [reply]
Well, I was able to get rid of a lot of the space at the top, but it made the glyphs wider to where it doesn't fit in the allocated space.  Guess I just have to find a different font.  :(
